.github/workflows/pull-request-review.yml

```
@@ -0,0 +1,46 @@
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            // List all reviews for the PR
+            const { data: reviews } = await github.rest.pulls.listReviews({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.payload.pull_request.number
+            });
+
+            // Filter reviews that have a user login containing "coderabbit" (case-insensitive)
+            const codeRabbitReviews = reviews.filter(review => 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)